src/concat.js

```diff
@@ -13,8 +13,8 @@ var invoker = require('./invoker');
  * @category List
  * @sig [a] -> [a] -> [a]
  * @sig String -> String -> String
- * @param {Array|String} a
- * @param {Array|String} b
+ * @param {Array|String|Object} a
+ * @param {Array|String|Object} b
```

Comment:
Correct (just without the trailing `a` in the `@return` line). It would be nice if JSDoc allowed us to indicate that the other types should be the same as whatever is chosen for the first param, but we'll have to rely on our `@sig` tags for that.